src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -130,8 +134,10 @@ $pf-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl"
   --pf-c-toolbar--spacer: var(--pf-c-toolbar__group--spacer);
 
   display: flex;
+  flex-wrap: wrap;
```

Comment:
This seems like a pretty big change, we don't think this will be breaking? row-gap or not, seems like it will change the way items lay out in the toolbar. And some people won't have `row-gap` yet either.

Reply:
You're right, we already have `pf-m-wrap`.